src/openrct2-ui/windows/GuestList.cpp

```diff
@@ -482,29 +518,14 @@ static void window_guest_list_update(rct_window* w)
  */
 static void window_guest_list_scrollgetsize(rct_window* w, int32_t scrollIndex, int32_t* width, int32_t* height)
 {
-    int32_t i, y, numGuests, spriteIndex;
-    Peep* peep;
-
+    int32_t i, y;
```

Comment:
Declaration of `i` can be moved to the assignment. And you might want to initialise `y` to 0.